packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -20,7 +20,8 @@ export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
         flexDirection: 'row',
         flexWrap: 'nowrap',
         alignItems: 'stretch',
-        padding: '0 0 0 8px',
+        // The 1px top and bottom padding ensure the input field does not overlap the border
```

Comment:
Just confirming--these get RTL-ified automatically, right?

Reply:
I'm 90% sure js styling does handle this automagically. I'll confirm.

Reply:
yes, RTL is automatic